spec/support/go_mod.rb

```diff
@@ -0,0 +1,42 @@
+class GoMod
+  attr_reader :path, :go_version, :requires
+
+  def initialize(path)
+    require 'pathname'
+    @path     = Pathname.new(path)
+    @requires = {}
+  end
+
+  def parse
+    return if @parsed
+    parse_block = nil
+
+    path.read.each_line do |line|
+      if line.start_with?("go ")
+        parse_go_version(line)
+        next
+      elsif line.start_with?("require (")
+        parse_block = :require
+      elsif line.start_with?(")")
```

Comment:
Same here, could we have spaces before the paren which would break this?

Reply:
Removed as discussed